lib/jsdom/living/nodes/SVGElement.idl

```diff
@@ -0,0 +1,6 @@
+interface SVGElement : Element {
```

Comment:
The changes are mostly additive. I believe there is a general desire to make SVG elements more like HTML elements where possible.